### ISSUE TITLE
ikart gazebo 

### ISSUE THREAD

**Issue by amaroyo** [robotology/ikart#8]
Hello @Tobias-Fischer, 

I heard you were interested in a gazebo simulator for the ikart, how far did you go regarding this?

Regards, 
Alex.

**Comment by Tobias-Fischer** [comment]
Hi @amaroyo,
Unfortunately I never really went further with this. For my purposes, the ROS/rviz integration of the `ikart` was sufficient.
Please let me know though if you are working on it.

Best,
Tobias

**Comment by amaroyo** [comment]
Yes, I am creating an ikart model for the gazebo simulator! I just started few days ago and @randaz81 is helping me out!

**Comment by Tobias-Fischer** [comment]
That sounds great! Please let me know in case you run into issues, I might be able to help.

Thanks,
Tobi

**Comment by traversaro** [comment]
Note that we have the inertial parameters from the CAD for iKart in here : https://github.com/robotology/ikart/issues/4 .

**Comment by Tobias-Fischer** [comment]
Hi @amaroyo and @randaz81,
I just wanted to hear what the current status on this is. I got interested in this topic again, and would be happy to help out. Thus far, I was able to spawn the `iKart` in `Gazebo`. However, I am not able to control it using `yarpmotorgui` (at least not in position mode, but only in torque mode). Furthermore, the `iKartRosBridge` cannot run because the required laser / odometry / odometer ports do not exist.
Would be great to hear what you are working on and how I could kick in.

Best,
Tobi

**Comment by Tobias-Fischer** [comment]
Any news on this @randaz81? Especially on how to use the ROS navigation stack with the iKart Gazebo stuff?

**Comment by Tobias-Fischer** [comment]
Did this integration ever happen @randaz81 @amaroyo?

**Comment by amaroyo** [comment]
Lately I haven't done anything on the ikart.. maybe @randaz81 can give you a better answer.. sorry!
